lib/util/xml_utils.js

```diff
@@ -228,3 +228,22 @@ shaka.util.XmlUtils.parseFloat = function(floatString) {
 };
 
 
+/**
+ * Evaluate a division expressed as a string
+ * @param {string} exprString
+ *   The expression to evaluate, e.g. "200/2". Can also be a single number
+ * @return {?number} The evaluated expression as floating point number on
+ *   success; otherwise return null.
+ */
+shaka.util.XmlUtils.evalDivision = function(exprString) {
+  var res;
+  var n;
+  if (res = exprString.match(/^(\d+)\/(\d+)$/)) {
+    n = Number(res[1] / res[2]);
+  } else if (res = exprString.match(/^([0-9]\.)$/)) {
```

Comment:
With this match, Number(res[1]) is the same as Number(exprString), so I would just drop this else if completely.